tests/network/vmi_multus.go

```diff
@@ -1143,35 +1143,38 @@ var _ = SIGDescribe("[Serial]Macvtap", func() {
 			libvmi.WithNetwork(libvmi.MultusNetwork(macvtapNetworkName)))
 	}
 
+	createVMIAndWaitReady := func(vmi *v1.VirtualMachineInstance, timeout int) *v1.VirtualMachineInstance{
+		vmi = tests.RunVMIAndExpectLaunchWithIgnoreWarningArg(vmi, timeout, false)
+		// Ensure QEMU GuestAgent is up as it should report network interfaces stats
+		tests.WaitAgentConnected(virtClient, vmi)
+		// Fetch latest VMI state
+		vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		return vmi
+	}
+
 	createCirrosVMIStaticIPOnNode := func(nodeName string, networkName string, ifaceName string, ipCIDR string, mac *string) *v1.VirtualMachineInstance {
 		var vmi *v1.VirtualMachineInstance
 		if mac != nil {
 			vmi = newCirrosVMIWithExplicitMac(networkName, *mac)
 		} else {
 			vmi = newCirrosVMIWithMacvtapNetwork(networkName)
 		}
-		vmi = tests.WaitUntilVMIReady(
-			tests.StartVmOnNode(vmi, nodeName),
-			console.LoginToCirros)
+		vmi = createVMIAndWaitReady(vmi, 360)
+		Expect(console.LoginToCirros(vmi)).ToNot(HaveOccurred())
```

Comment:
you sure we have guest agent on Cirros ? not sure
(createVMIAndWaitReady waits for it)

Reply:
No, good catch!
We dont have it on cirrors, I will remove it.